libraries/Microsoft.Bot.Builder.Dialogs.Debugging/Protocol/Scopes.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Dialogs.Debugging.Protocol
+{
+#pragma warning disable CA1724 // Type names should not match namespaces (by design and we can't change it)
```

Comment:
remove

Reply:
Leaving it for now, we can change it when we make it public.